server/pulp/server/managers/content/upload.py

```diff
@@ -207,6 +208,13 @@ def import_uploaded_unit(repo_id, unit_type_id, unit_key, unit_metadata, upload_
         try:
             result = importer_instance.upload_unit(transfer_repo, unit_type_id, unit_key,
                                                    unit_metadata, file_path, conduit, call_config)
+            if not result['success_flag']:
```

Comment:
Unfortunately, I noticed that at least one plugin (``pulp_docker``) does not return a dictionary as it should. It return ``None``. That's been ok so far, because nothing has ever checked the value of ``result`` here or down the stack.

One option is to just fix pulp_docker, and check the other plugins to make sure they return the right thing.

Another option is to be defensive, and skip this check in case ``result`` is the wrong type or doesn't have that attribute.